common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -48,6 +48,16 @@ trait ABTestSwitches {
 
   Switch(
     ABTests,
+    "ab-contributions-story-20160922",
```

Comment:
Have you read our naming guidelines in our [ab testing doc](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/01-ab-testing.md)? 

> The hyphen-separated id that follows ab- must correspond to the TitleCased id defined in the JS test module. e.g. if the switch id is ab-geo-most-popular, the test id must be GeoMostPopular


Also, read the section on ab test id